guides/doc-Upgrading_and_Updating/topics/migrating_pulp_content.adoc

```diff
@@ -0,0 +1,70 @@
+[id="migrating_pulp_content"]
+= Migrating Content to Pulp 3
+
+The time to prepare content for Pulp 3 depends on the amount of content and the number of content views.
+For large systems, this can mean several days of downtime.
+To prevent this, {Team} recommends that you pre-migrate Pulp content while running the latest version of {ProjectServer} {ProjectVersionPrevious}.
```

Comment:
I am trying to find more info about pre-migration without success.

Reply:
A comment from the original google docs draft oif this document:
Eric Helms
7:31 PM Aug 16
Do we want to say pre-migrate instead of migrate? I don't want users to think that within 6.9 their content is fully migrated to and being served by Pulp 3.